hack/generate/samples/helm/memcached.go

```diff
@@ -23,8 +23,10 @@ import (
 
 	"github.com/operator-framework/operator-sdk/hack/generate/samples/pkg"
 	"github.com/operator-framework/operator-sdk/test/utils"
+	testutils "github.com/operator-framework/operator-sdk/test/utils"
```

Comment:
The changes in this file are only to add better comments, fix log error type msg and keeping all following the same standard.